Milestone 1/Main Page.html

```diff
@@ -0,0 +1,59 @@
+<!-- Milestone 1 -->
```

Comment:
When working with HTML, it's best practice to name your homepage `index.html`. Also in general, it's best to avoid spaces in file names, I usually just put a dash where a space should be.